docker/ci/Dockerfile

```diff
@@ -9,24 +9,14 @@ RUN apt-get update -qq && apt-get install -yqq \
     g++ \
```

Comment:
Looks like we need Python as well:

```
-- Could NOT find PythonInterp (missing:  PYTHON_EXECUTABLE) 
-- Configuring incomplete, errors occurred!
```

Reply:
Boost is also installed in a slightly different location now so cannot be found.

In ci/build-travis.sh it uses cmake -DBOOST_ROOT=/usr/local, but bootstrap_boost.sh installs it in /usr/local/**boost**

Reply:
The environment variable BOOST_ROOT, if set, will override where `bootstrap_boost.sh` puts the boost install.

Reply:
> The environment variable `BOOST_ROOT`, if set, will override where `bootstrap_boost.sh` puts the boost install.

Didn't know that, thanks! I did try changing build-travis.sh to use /usr/local/boost which worked ok, but tried your suggestion instead which also works, and that is what I have now done.

There are however some warnings "New Boost version may have incorrect or missing dependencies and imported targets". I think this is because it is now using boost 1.69 instead of 1.66 but our version of FindBoost.cmake has not been updated to support it. I don't want to extend the scope of this task more than I need to so am using additional arguments "**-B 1.66**" to bootstrap_boost.sh to use the version of Boost that Docker was originally using before.

> Looks like we need Python as well:
> 
> ```
> -- Could NOT find PythonInterp (missing:  PYTHON_EXECUTABLE) 
> -- Configuring incomplete, errors occurred!
> ```
Actually the missing python message is an existing error in the travis-ci Linux build, but I have modified the Dockerfile to install python to resolve it.

Reply:
Thanks !